src/extensions/FileUploadExt.ts

```
@@ -0,0 +1,2205 @@
+              url = result.data.url;
+            }
+          }
+          
+          resolve(url);
+        } catch (error) {
+          reject(error);
+        }
+      });
+    },
+
+    // New method to process image uploads
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)